lib/data/data_sources/remote/api_service.dart

```diff
@@ -0,0 +1,46 @@
+
+import 'package:http/http.dart';
+import '../../../domain/repository/service/i_movie_service.dart';
+import 'api_const.dart';
+
+class APIMovieServiceImpl implements MovieService {
+
+  const APIMovieServiceImpl();
+
+
+  @override
+  Future<Response> getNowPlayingMovies() async => get(
+        Uri.parse(
+          '$API_BASE_URL/$API_THIRD_VERSION_PATH/$API_MOVIE_PATH/$API_NOW_PLAYING_PATH?$API_KEY',
```

Comment:
If you notice it this baseUrl that you are using (`$API_BASE_URL/$API_THIRD_VERSION_PATH/$API_MOVIE_PATH/`) is the same for all the methods

Reply:
done